package.json

```diff
@@ -10,12 +10,22 @@
     "asciidoctor-pdf": "bin/asciidoctor-pdf",
     "asciidoctor-web-pdf": "bin/asciidoctor-web-pdf"
   },
+  "pkg": {
+    "assets": [
+      "css/**/*.css",
+      "node_modules/@fortawesome/fontawesome-svg-core/*.css"
+    ],
+    "scripts": [
+      "lib/document/**/*.js"
+    ]
+  },
   "files": [
     "bin",
     "lib",
     "css"
   ],
   "scripts": {
+    "build": "rm -rf build && node tasks/prepare_binaries.js && ./tasks/prepare_binaries_zip.sh",
```

Comment:
(suggestion): We should probably do everything using Node libraries, it will be more portable. What do you think?

Reply:
Sure, I'll have a go. Originally I didn't realise we were using `fs-extra` and the horror of using pure Node was too much for me.

Reply:
No worries, if you are not comfortable with this part I can probably take care of it.